thrift/lib/cpp/transport/TFileTransport.cpp

```diff
@@ -424,7 +424,7 @@ void TFileTransport::writerThread() {
             offset_ = lseek(fd_, 0, SEEK_CUR);
             int32_t padding = (int32_t)((offset_ / chunkSize_ + 1) * chunkSize_ - offset_);
 
-            uint8_t zeros[padding];
+            uint8_t* zeros = (uint8_t*)alloca(sizeof(uint8_t) * padding);
```

Comment:
`alloca` is not a standard function, not even in POSIX. Besides, `alloca` may fail.
We need a different solution here.